conf/application.conf

```
@@ -146,6 +146,13 @@ schedules {
     enabled = false
     mongoLockTimeout = 7200 # 2 hours in seconds used to release the mongo lock in case the job fails with exception and doesn't unlock
   }
+
+  NotProcessedFilesService {
+    description = "Job to set files recieved SDES that haven't updated to NOT_PROCESSED_PENDING_RETRY"
```

Bit more description needed:
```suggestion
    description = "Job to set retry file notifications where FileProcessed has not been received by SDES"
```